### ISSUE TITLE
I would like to be able to resolve an issue to a commit that hasn't been deployed

### ISSUE THREAD

**Issue by eric** [getsentry/sentry#41999]
### Problem Statement

When I have finished fixing something, I would like to go into Sentry and specify the commit that has fixed it even if it hasn't been deployed yet. Right now I have to wait until it's been deployed and then remember to go resolve it.

### Solution Brainstorm

It would be nice if you could just store the SHA or check with GitHub to see if it's valid.

**Comment by brianthi** [comment]
Hi @eric, if you have the Github integration installed, you can follow instructions here on how to achieve this when committing your code: https://docs.sentry.io/product/releases/associate-commits/#resolve-issues-by-commit.  Let us know if this solves the problem you're running into.

**Comment by eric** [comment]
I would like to be able to do this from the Sentry UI if I don’t do it in a commit message

**Comment by eric** [comment]
I still would like to do this.

**Comment by armenzg** [comment]
I see you're talking about this functionality. I can see that I cannot use a SHA from master.
<img width="273" alt="image" src="https://user-images.githubusercontent.com/44410/213489134-187d1189-9480-401e-ba46-a9dfae9f05d6.png">

<img width="520" alt="image" src="https://user-images.githubusercontent.com/44410/213489068-3fb4ab89-46bc-4d31-8fd3-1d72095e0032.png">

**Comment by gaprl** [comment]
Hey @eric, this issue was opened over 2 years ago. Is this still a feature you'd find valuable? If we don't hear back in the next couple of weeks, we'll close this issue.

**Comment by eric** [comment]
Yes it would still be useful to me. It is a workflow that I would use if it were available.
